Combinatorial_map/include/CGAL/Combinatorial_map_iterators_base.h

```diff
@@ -389,7 +389,7 @@ namespace CGAL {
     {}
 
     /// Destructor.
-    ~CMap_non_basic_iterator() CGAL_NOEXCEPT(CGAL_NO_ASSERTIONS_BOOL)
+    ~CMap_non_basic_iterator() noexcept(!CGAL_assertions)
```

Comment:
This essentially renames CGAL_NO_ASSERTIONS_BOOL. Did you not like the name?
The lower-case name makes the difference with CGAL_assertion not very clear.

Reply:
I thought it was maybe less ugly. ALL CAPS seems more reminiscent of C89 based code.

You make a valid point about it being harder to distinguish between `CGAL_assertion` and `CGAL_assertions`. I also considered `CGAL_assertions_enabled`

Reply:
The CGAL guidelines mandate that macros names must be uppercase. If the code looks old, that is because it uses a macro, and not because of the naming guidelines.

Reply:
@lrineau Well I know, since I added CGAL_NO_ASSERTIONS_BOOL in https://github.com/CGAL/cgal/commit/f0ac2367139bc8190816754471e382e429d064f1

It's just that for the other assertion macros the code allows lowercase i.e `CGAL_assertion_msg` etc...
Anyway, is something like `CGAL_ASSERTIONS_ENABLED` preferred, or should I just revert back to the old code and add `CGAL_NO_PRECONDITONS_BOOL`